autogpt_platform/backend/backend/blocks/code_executor_test.py

```
@@ -134,6 +134,16 @@ def test_oversized_payload_raises(self):
         with pytest.raises(ValueError, match="too large"):
             build_variable_injection(big, ProgrammingLanguage.PYTHON)
 
+    def test_emoji_in_variables_is_safe_for_env_var(self):
+        """Emoji (and any Unicode) should serialize without surrogate errors."""
+        variables = {"message": "Holidays 🎅🏻", "name": "João"}
+        envs, prefix = build_variable_injection(variables, ProgrammingLanguage.PYTHON)
```

Ruff (0.16.2)

[warning] Unpacked variable `prefix` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)